homeassistant/components/binary_sensor/mychevy.py

```diff
@@ -0,0 +1,68 @@
+"""Support for MyChevy sensors."""
+
+import asyncio
+from logging import getLogger
+
+from homeassistant.components.mychevy import (
+    EVBinarySensorConfig, DOMAIN,
+)
+from homeassistant.components.binary_sensor import (
+    ENTITY_ID_FORMAT, BinarySensorDevice)
+from homeassistant.util import (slugify)
+
+_LOGGER = getLogger(__name__)
+
+SENSORS = [
+    EVBinarySensorConfig("Plugged In", "plugged_in", "plug")
+]
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
```

Comment:
Since no I/O is performed in this function you can switch to `async_setup_platform`.